projects/swell-vault/index.js

```diff
@@ -10,5 +10,6 @@ module.exports = {
             ADDRESSES.ethereum.WETH,
             ADDRESSES.ethereum.STETH
         ]}),
+        doublecounted: true, //
```

Comment:
dont think this is doublecounted? though it is weird that same contract address is used by two different protocols (but different chain)